metrics/disk/cassandra_kubernetes/cassandra.sh

```diff
@@ -125,21 +125,51 @@ function cassandra_start() {
 	wait_time=20
  	sleep_time=2
 
+	vol_capacity="500M"
+	volume_name="block-loop-pv"
+	volume_claim="block-loop-pvc"
+
+	# Create Loop Device
+	export tmp_disk_image=$(mktemp --tmpdir disk.XXXXXX.img)
+	truncate "$tmp_disk_image" --size "$vol_capacity"
+	export loop_dev=$(sudo losetup -f)
+	sudo losetup "$loop_dev" "$tmp_disk_image"
+
+	# Create Storage Class
+	kubectl create -f "${SCRIPT_PATH}/volume/local-storage.yaml"
+
+	# Create Persistent Volume
+	export tmp_pv_yaml=$(mktemp --tmpdir block_persistent_vol.XXXXX.yaml)
+	sed -e "s|LOOP_DEVICE|${loop_dev}|" "${SCRIPT_PATH}/volume/block-loop-pv.yaml" > "$tmp_pv_yaml"
+	sed -i "s|HOSTNAME|$(hostname | awk '{print tolower($0)}')|" "$tmp_pv_yaml"
+	sed -i "s|CAPACITY|${vol_capacity}|" "$tmp_pv_yaml"
+
+	kubectl create -f "$tmp_pv_yaml"
+	cmd="kubectl get pv/${volume_name} | grep Available"
+	waitForProcess "$wait_time" "$sleep_time" "$cmd"
+
+	# Create Persistent Volume Claim
+	export tmp_pvc_yaml=$(mktemp --tmpdir block_persistent_vol.XXXXX.yaml)
+	sed -e "s|CAPACITY|${vol_capacity}|" "${SCRIPT_PATH}/volume/block-loop-pvc.yaml" > "$tmp_pvc_yaml"
+	kubectl create -f "$tmp_pvc_yaml"
+
 	# Create service
 	kubectl create -f "${SCRIPT_PATH}/runtimeclass_workloads/cassandra-service.yaml"
 
 	# Check service
 	kubectl get svc | grep "$service_name"
 
-	# Create local volumes
-	kubectl create -f "${SCRIPT_PATH}/runtimeclass_workloads/local-volumes.yaml"
-
-	# Create statefulset
-	kubectl create -f "${SCRIPT_PATH}/runtimeclass_workloads/cassandra-statefulset.yaml"
```

Comment:
Why is the stateful set not being used anymore?

Reply:
@amshinde yes it is being used see https://github.com/kata-containers/tests/blob/9822556eb79afac3f8f1ca4bbe169850dc3b10f6/metrics/disk/cassandra_kubernetes/cassandra.sh#L165